<Type Name="XamlSchemaException" FullName="System.Xaml.XamlSchemaException">
  <Metadata><Meta Name="ms.openlocfilehash" Value="a62dd6b3261a7acbed5a1bf5a1cf79945a4a29b5" /><Meta Name="ms.sourcegitcommit" Value="c0c07dbd19cd7017243f9ac36915755f79bc8da6" /><Meta Name="ms.translationtype" Value="MT" /><Meta Name="ms.contentlocale" Value="es-ES" /><Meta Name="ms.lasthandoff" Value="11/27/2018" /><Meta Name="ms.locfileid" Value="52357677" /></Metadata><TypeSignature Language="C#" Value="public class XamlSchemaException : System.Xaml.XamlException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit XamlSchemaException extends System.Xaml.XamlException" />
  <TypeSignature Language="DocId" Value="T:System.Xaml.XamlSchemaException" />
  <TypeSignature Language="VB.NET" Value="Public Class XamlSchemaException&#xA;Inherits XamlException" />
  <TypeSignature Language="C++ CLI" Value="public ref class XamlSchemaException : System::Xaml::XamlException" />
  <TypeSignature Language="F#" Value="type XamlSchemaException = class&#xA;    inherit XamlException" />
  <AssemblyInfo>
    <AssemblyName>System.Xaml</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Xaml.XamlException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
      <AttributeName>System.Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="d1542-101">Excepción que se produce cuando un sistema de enlace u otro sistema de representación de esquema XAML notifica una excepción al contexto de esquema.</span><span class="sxs-lookup"><span data-stu-id="d1542-101">The exception that is thrown when a binding system or another schema representation system for XAML reports an exception to the schema context.</span></span></summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="d1542-102">El <xref:System.Xaml.XamlSchemaException> produce varios casos en que las clases de compatibilidad de enlace de CLR que proporcionan el contexto de esquema para los servicios XAML de .NET Framework (<xref:System.Xaml.XamlXmlReader>, <xref:System.Xaml.XamlObjectWriter>, o <xref:System.Xaml.XamlServices>).</span><span class="sxs-lookup"><span data-stu-id="d1542-102">The <xref:System.Xaml.XamlSchemaException> is thrown by several cases in the CLR binding support classes that provide the schema context for .NET Framework XAML Services (<xref:System.Xaml.XamlXmlReader>, <xref:System.Xaml.XamlObjectWriter>, or <xref:System.Xaml.XamlServices>).</span></span> <span data-ttu-id="d1542-103">Un <xref:System.Xaml.XamlSchemaException> normalmente se pasa como la excepción interna para otras excepciones, como el <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=nameWithType>-true la propiedad para un <xref:System.Xaml.XamlObjectWriterException>, por lo que debería ser poco habitual para detectar un <xref:System.Xaml.XamlSchemaException> directamente a menos que se va a implementar un componente como un escritor de objetos especializada.</span><span class="sxs-lookup"><span data-stu-id="d1542-103">A <xref:System.Xaml.XamlSchemaException> is typically passed as the inner exception for other exceptions, such as the <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=nameWithType>-true property for a <xref:System.Xaml.XamlObjectWriterException>, so it should be uncommon to catch a <xref:System.Xaml.XamlSchemaException> directly unless you are implementing a component such as a specialized object writer.</span></span>  
  
 ]]></format>
    </remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="d1542-104">Inicializa una nueva instancia de la clase <see cref="T:System.Xaml.XamlSchemaException" />.</span><span class="sxs-lookup"><span data-stu-id="d1542-104">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="d1542-105">Inicializa una nueva instancia de la clase <see cref="T:System.Xaml.XamlSchemaException" /> con un mensaje proporcionado por el sistema que describe el error.</span><span class="sxs-lookup"><span data-stu-id="d1542-105">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a system-supplied message that describes the error.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : string -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="d1542-106">Mensaje que describe la excepción.</span><span class="sxs-lookup"><span data-stu-id="d1542-106">The message that describes the exception.</span></span> <span data-ttu-id="d1542-107">El autor de llamada de este constructor debe asegurarse de que la cadena se haya adaptado para la referencia cultural actual del sistema.</span><span class="sxs-lookup"><span data-stu-id="d1542-107">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span></param>
        <summary><span data-ttu-id="d1542-108">Inicializa una nueva instancia de la clase <see cref="T:System.Xaml.XamlSchemaException" /> con un mensaje especificado que describe el error.</span><span class="sxs-lookup"><span data-stu-id="d1542-108">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a specified message that describes the error.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected XamlSchemaException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; XamlSchemaException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : System.Runtime.Serialization.SerializationInfo * System.Runtime.Serialization.StreamingContext -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException (info, context)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info"><span data-ttu-id="d1542-109">Objeto que contiene los datos del objeto serializados.</span><span class="sxs-lookup"><span data-stu-id="d1542-109">The object that holds the serialized object data.</span></span></param>
        <param name="context"><span data-ttu-id="d1542-110">Información contextual acerca del origen o del destino.</span><span class="sxs-lookup"><span data-stu-id="d1542-110">The contextual information about the source or destination.</span></span></param>
        <summary><span data-ttu-id="d1542-111">Inicializa una nueva instancia de la clase <see cref="T:System.Xaml.XamlSchemaException" /> con datos serializados.</span><span class="sxs-lookup"><span data-stu-id="d1542-111">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with serialized data.</span></span></summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlSchemaException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlSchemaException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.Xaml.XamlSchemaException : string * Exception -&gt; System.Xaml.XamlSchemaException" Usage="new System.Xaml.XamlSchemaException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Runtime.TargetedPatchingOptOut("Performance critical to inline this type of method across NGen image boundaries")</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="d1542-112">Mensaje que describe la excepción.</span><span class="sxs-lookup"><span data-stu-id="d1542-112">The message that describes the exception.</span></span> <span data-ttu-id="d1542-113">El autor de llamada de este constructor debe asegurarse de que la cadena se haya adaptado para la referencia cultural actual del sistema.</span><span class="sxs-lookup"><span data-stu-id="d1542-113">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span></param>
        <param name="innerException"><span data-ttu-id="d1542-114">Excepción que es la causa de la excepción actual.</span><span class="sxs-lookup"><span data-stu-id="d1542-114">The exception that is the cause of the current exception.</span></span> <span data-ttu-id="d1542-115">Si el parámetro <paramref name="innerException" /> no es <see langword="null" />, la excepción actual se produce en un bloque <see langword="catch" /> que controla la excepción interna.</span><span class="sxs-lookup"><span data-stu-id="d1542-115">If the <paramref name="innerException" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="d1542-116">Inicializa una nueva instancia de la clase <see cref="T:System.Xaml.XamlSchemaException" /> con el mensaje de error especificado y una referencia a la excepción interna que representa la causa de esta excepción.</span><span class="sxs-lookup"><span data-stu-id="d1542-116">Initializes a new instance of the <see cref="T:System.Xaml.XamlSchemaException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="d1542-117">Una excepción que se produce como consecuencia directa de una excepción anterior debe incluir una referencia a esta última en la propiedad <xref:System.Exception.InnerException%2A>.</span><span class="sxs-lookup"><span data-stu-id="d1542-117">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="d1542-118">La propiedad <xref:System.Exception.InnerException%2A> devuelve el mismo valor que se haya pasado al constructor, o bien `null` en caso de que la propiedad <xref:System.Exception.InnerException%2A> no le haya proporcionado al constructor el valor de la excepción interna.</span><span class="sxs-lookup"><span data-stu-id="d1542-118">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>